app/assets/stylesheets/welcome.scss

```diff
@@ -114,15 +109,13 @@ iframe{
     display: block;
   }
 
-
-
-}
-
-@media screen and (min-width: 1024px) {
-  .categoryLabel {
-    font-size: 1.75em;
+  .instagram-feed{
+    grid-template-columns: 33% 33% 33%;
   }
 }
 
-
-
+@media screen and (min-width: 1200px){
```

Comment:
so I would normally disagree with the 1200px breakpoint here, but this lovely article strongly disagrees with me, so I will leave it here for you, and let you make your own call. Just keep them all consistent please:
https://medium.freecodecamp.com/the-100-correct-way-to-do-css-breakpoints-88d6a5ba1862